src/remapCoverage.ts

```diff
@@ -0,0 +1,94 @@
+import { red, blue, green, yellow } from 'chalk';
+import { accessSync, constants, unlinkSync } from 'fs';
+import * as path from 'path';
+import * as MemoryStore from 'istanbul/lib/store/memory';
+import { loadCoverage, remap, writeReport } from 'remap-istanbul/lib/main';
+import { TestArgs } from './main';
+
+const process: NodeJS.Process = require('process');
+const projectName = require(path.join(process.cwd(), './package.json')).name;
+
+/**
+ * Returns true for files that should be remapped.
+ *
+ * Matches files that don't start with `webpack:/` and end in `.js` (likely bundles)
+ * Or matches files that start with `webpack:///src/` but don't have an extension like `.css?`
+ */
+const REMAP_INCLUDE_PATTERN = /^(?:(?!webpack:\/).*\.js|webpack:\/{3}src\/(?!.*\.css\?))/;
+
+export default async function remapCoverage(testArgs: TestArgs) {
+
+	function checkCoverageFinal() {
+		try {
+			accessSync('coverage-final.json', constants.W_OK | constants.R_OK);
+		}
+		catch (e) {
+			return false;
+		}
+		return true;
+	}
+
+	if (testArgs.debug) {
+		console.log(blue.bold('\nRemapping coverage...\n'));
```

Comment:
Thinking about this some more, I think the option is actually a bit confusing (because I get coverage things but not all coverage things) and we should just do it all by default (and removes another configuration option yay!).